v0/destinations/ga/transform.js

```diff
@@ -299,11 +299,17 @@ function responseBuilderSimple(
       ? message.integrations[gaDisplayName].clientId
       : undefined
     : undefined;
-  finalPayload.cid =
-    integrationsClientId ||
-    getDestinationExternalID(message, "gaExternalId") ||
-    message.anonymousId ||
-    md5(message.userId);
+
+  const disableMd5 = get(message, "disableMd5");
```

Comment:
IMO, it's better to control it through a dashboard setting. WDYT @ruchiramoitra?

Reply:
yes. we can take it from destination config.

Reply:
Okay, update soon

Reply:
changes are updated